discoverd/testutil/testutil.go

```diff
@@ -1,87 +0,0 @@
-package testutil
-
-import (
-	"io"
-	"os"
-	"os/exec"
-
-	"github.com/flynn/flynn/discoverd/client"
-	. "github.com/flynn/flynn/discoverd/testutil/etcdrunner"
-)
-
-func RunDiscoverdServer(t TestingT, port string, etcdAddr string) (string, func()) {
```

Comment:
This is now [used in the discoverd client tests](https://github.com/flynn/flynn/blob/master/discoverd/client/watch_test.go#L22-L32), so we will need to keep it.